phpstan-baseline.neon

```diff
@@ -7729,3 +7729,13 @@ parameters:
 			message: "#^Method Sylius\\\\Component\\\\User\\\\Security\\\\Checker\\\\TokenUniquenessChecker\\:\\:__construct\\(\\) has parameter \\$repository with generic interface Sylius\\\\Component\\\\Resource\\\\Repository\\\\RepositoryInterface but does not specify its types\\: T$#"
 			count: 1
 			path: src/Sylius/Component/User/Security/Checker/TokenUniquenessChecker.php
+
+		-
+			message: "#^Method Sylius\\\\Component\\\\Core\\\\Promotion\\\\Applicator\\\\AdditionalFiltersApplicatorInterface\\:\\:apply\\(\\) has parameter \\$configuration with no value type specified in iterable type array\\.$#"
+			count: 1
+			path: src/Sylius/Component/Core/Promotion/Applicator/AdditionalFiltersApplicatorInterface.php
+
+		-
+			message: "#^Method Sylius\\\\Component\\\\Core\\\\Promotion\\\\Applicator\\\\AdditionalFiltersApplicator\\:\\:apply\\(\\) has parameter \\$configuration with no value type specified in iterable type array\\.$#"
+			count: 1
+			path: src/Sylius/Component/Core/Promotion/Applicator/AdditionalFiltersApplicator.php
```

Comment:
I think it's better to handle it with a proper type hinting. You can describe `$configuration` parameter in `AdditionalFiltersApplicatorInterface` with `@param array<string, mixed> $configuration`.

Reply:
nice suggestion! it's done